<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Who goes first?</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="computer_goes_first">
            <property name="text">
             <string>Computer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="human_goes_first">
            <property name="text">
             <string>Me</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="alternate_every_game">
            <property name="text">
             <string>Alternate every game</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Select Level</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="level_easy">
            <property name="text">
             <string>Easy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="level_intermediate">
            <property name="text">
             <string>Intermediate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="level_impossible">
            <property name="text">
             <string>Impossible</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="more_info">
          <property name="text">
           <string>More Information</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Visual Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Change Computer's...</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="computers_symbol">
               <property name="text">
                <string>Symbol</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="computers_color">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="computers_font">
               <property name="text">
                <string>Font</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Change Human's...</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QPushButton" name="humans_symbol">
               <property name="text">
                <string>Symbol</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="humans_color">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="humans_font">
               <property name="text">
                <string>Font</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="board_color">
            <property name="text">
             <string>Change Board Color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hint_blink_time">
            <property name="text">
             <string>Hint Blink Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>When loading a game use loaded game's...</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QCheckBox" name="ld_symbols">
          <property name="text">
           <string>Symbols</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ld_engine_settings">
          <property name="text">
           <string>Engine Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ld_visual_settings">
          <property name="text">
           <string>Visual Settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </item>
   <item>
    <widget class="QPushButton" name="engine_opts">
     <property name="text">
      <string>Change Engine/Game Options</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="save_settings">
       <property name="text">
        <string>Export Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_settings">
       <property name="text">
        <string>Load Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="convert_to_normal">
     <property name="text">
      <string>Use these as normal settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
